Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/webapp/WebAppPropertyViewPresenter.java

```diff
@@ -49,8 +49,8 @@ protected boolean getPublishingProfile(@NotNull final String sid, @NotNull final
     }
 
     @Override
-    protected WebAppBase getWebApp(@NotNull final String sid, @NotNull final String webAppId,
-                                   @Nullable final String name) throws Exception {
+    protected WebAppBase getWebAppBase(@NotNull final String sid, @NotNull final String webAppId,
+                                       @Nullable final String name) throws Exception {
```

Comment:
remove the extra spaces

Reply:
Those spaces are used to align the parameters.